lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -1112,36 +1112,42 @@ def set_search_path(dbapi_connection, connection_record):
 Several options for CREATE TABLE are supported directly by the PostgreSQL
 dialect in conjunction with the :class:`_schema.Table` construct:
 
-* ``TABLESPACE``::
+* ``INHERITS``::
 
-    Table("some_table", metadata, ..., postgresql_tablespace='some_tablespace')
+    Table("some_table", metadata, ..., postgresql_inherits="some_supertable")
 
-  The above option is also available on the :class:`.Index` construct.
+    Table("some_table", metadata, ..., postgresql_inherits=("t1", "t2", ...))
 
 * ``ON COMMIT``::
 
     Table("some_table", metadata, ..., postgresql_on_commit='PRESERVE ROWS')
 
-* ``WITH OIDS``::
+* ``PARTITION BY``::
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5114